packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -263,6 +263,11 @@ abstract class FlutterCommand extends Command<void> {
       help: 'Enables expression evaluation in the debugger.',
       hide: !verboseHelp,
     );
+    argParser.addOption('web-launch-url',
+      help: 'The launch URL the browser should use preferentially. If not provided, '
+          'the default launch URL depends on "--web-hostname" and "--web-port".',
+      hide: !verboseHelp,
```

Comment:
This seems pretty useful for normal users, why would we hide it?

Reply:
yes, you are right.